### PR TITLE
Fixing maintenance mode link in BO header

### PR DESCRIPTION
**Pull request by Poulinhoo**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixing the maintenance mode display on BO.
| Type?             | bug fix 
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| UI Tests | https://github.com/florine2623/testing_pr/actions/runs/10578141554
| How to test?      | Follow the steps on this [issue](https://github.com/PrestaShop/PrestaShop/issues/36599)
| Fixed issue or discussion?     | Fixes #[36599](https://github.com/PrestaShop/PrestaShop/issues/36599)

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @PoulainMaxime!

This is your first pull request on [PrestaShop](https://github.com/PrestaShop/PrestaShop/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Review by Hlavtox — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @PoulainMaxime ,

I'm not sure the color orange is in phase with the new design.

Screenshot 2024-08-27 at 14 55 19

What do you think @tblivet ?

File: src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig [hunk 1]
```diff
@@ -113,7 +113,9 @@
              title="{{ maintenanceTitle|e('html') }}"
              href="{{ url('admin_maintenance') }}"
           >
-            <i class="material-icons">build</i>
+            <i class="material-icons"
+               style="{% if ps.frontOfficeAccessibleForAdmins|default(false) %}color: var(--green);{% endif %}"
```
Comment (tblivet) [inline src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig]: Hi 👋 In agreement with the designers, the link will be in orange (already handled by CSS), so we only need `<i class="material-icons">build</i>`. The green color and condition are no longer needed.

**Review by tblivet — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

**Review by ShaiMagal — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig (hunk 1 above)
Comment (jolelievre) [reply src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig]: @PoulainMaxime you should remove this extra style, the design team considered it useless and there is no more highlight related to `frontOfficeAccessibleForAdmins` anymore

However your fix about the variable `isMaintenanceEnabled` seems legit and should fix the initial bug. Can you please clean this style part and then the PR should be testable

**Review by jolelievre — CHANGES_REQUESTED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @PoulainMaxime

**Review by boherm — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @PoulainMaxime ,

Tested all features on maintenance page.
Checked the link on migrated and legacy pages.
The link in header for maintenance mode is well displayed.

LGTM ✅ 
Thanks!

**Comment by nicosomb** [comment]
Thank you @m-Poulain and @florine2623 !

**Comment by nicosomb** [comment]
@PoulainMaxime congratulations for your first merged pull request!

**Pull request merged**